### PR TITLE
Added is_deleted to contact filters

### PR DESCRIPTION
**Pull request by am2605**
Suggested fix for #295

### CODE REVIEW

**Comment by eileenmcnaughton** [comment]
hmm - it triggered an enotice on the test https://test.civicrm.org/job/Extension-SHA/614/testReport/junit/(root)/api_v3_ReportTemplate_GetmetadataTest/testApiMetadata/

**Comment by francescbassas** [comment]
We tested on local development and this PR seems solve the described issue. Could it be reopened and re-tested so we could check if it still gives the enotice?

**Comment by eileenmcnaughton** [comment]
@francescbassas done - other than the test issue I don't see any problem with this

**Comment by eileenmcnaughton** [comment]
I made a change to the test - it has a whitelist of fields where there is a field but no filter - I think it can pass now

File: CRM/Extendedreport/Form/Report/ColumnDefinitionTrait.php [hunk 1]
```diff
@@ -326,6 +326,15 @@ function getContactColumns($options = []) {
         'is_filters' => TRUE,
         'is_contact_filter' => TRUE,
       ],
+      $options['prefix'] . 'is_deleted' => [
+        'title' => $options['prefix_label'] . ts('Is deleted'),
```
[the remark this answers is not in the document]
Comment (seamuslee001) [reply CRM/Extendedreport/Form/Report/ColumnDefinitionTrait.php:330]: Probably wouldn't the better option be 

```suggestion
        'title' => E::ts('%1 'Is deleted', [1 => $options['prefix_label']),
```

**Comment by francescbassas** [comment]
Nice @eileenmcnaughton! I understand that this will be released with 5.5.

Thanks for this amazing an powerful extension.

**Pull request merged**